### PR TITLE
[storage] Fixed downloading of an empty routing file.

### PR DESCRIPTION
**Pull request by ygorshenin**
https://trello.com/c/aP8ysG4s/2017-south-georgia-and-the-south-sandwich-islands

### CODE REVIEW

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

File: storage/storage.cpp [hunk 1]
```diff
@@ -734,7 +736,15 @@ MapOptions Storage::NormalizeDownloadFileSet(TIndex const & index, MapOptions op
     {
       opt = UnsetOptions(opt, file);
     }
+
+    // Check whether requested file is not empty.
+    if (GetRemoteSize(country, file) == 0)
+    {
+      ASSERT_NOT_EQUAL(MapOptions::Map, file, ("Map can't be empty."));
+      opt = UnsetOptions(opt, file);
```
Comment (syershov) [inline storage/storage.cpp]: В этой функции file и opt надо переименовать в option & options.

Comment (ygorshenin) [inline storage/storage.cpp]: Done

**Comment by ygorshenin** [comment]
PTAL

**Comment by syershov** [comment]
LGTM

**Pull request merged**